packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -556,7 +552,9 @@ class AnalyzeCommand extends FlutterCommand {
 
       if (firstAnalysis && _isBenchmarking) {
         _writeBenchmark(analysisTimer, issueCount);
-        exit(0);
+        server.dispose().then((dynamic result) {
```

Comment:
FWIW, in general I recommend avoiding `_` in Flutter code because it makes it harder to copy-paste the code (because when you copy it, you don't immediately know the type of the argument you're copying, and so it's harder to look it up).